cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -827,32 +827,43 @@ def test_fsx_architecture_os_validator(architecture, os, expected_message):
 
 
 @pytest.mark.parametrize(
-    "mount_dir_list, expected_message",
+    "shared_mount_dir_list, local_mount_dir_list, expected_message",
     [
         (
             ["dir1"],
+            [],
             None,
         ),
         (
             ["dir1", "dir2"],
+            [],
             None,
         ),
         (
             ["dir1", "dir2", "dir3"],
+            [],
             None,
         ),
         (
             ["dir1", "dir1", "dir2"],
+            [],
             "Mount directory dir1 cannot be specified for multiple file systems",
         ),
         (
             ["dir1", "dir2", "dir3", "dir2", "dir1"],
+            [],
             "Mount directories dir2, dir1 cannot be specified for multiple file systems",
         ),
+        (
+            ["dir1", "dir2", "/scratch"],
```

Comment:
Since `/scratch` is a notable mount point, I think it's worth it to have a test case verifying that when shared_mount_dir_list contains `/shared` and local mount dir does not, the validator succeeds

Reply:
I included the test case in integration test.

Reply:
I think we should add it in unit tests because:
1. validator logic is expected to be covered by unit tests 
2. this is a notable test case for the validator
3. if something can be tested both with unit tests and integ test, better to have in unit test at forst because they are faster and cheaper

Reply:
Done